gce_workload_certs_refresh/main.go

```diff
@@ -0,0 +1,290 @@
+//  Copyright 2022 Google LLC
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// GoogleAuthorizedKeys obtains SSH keys from metadata.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/GoogleCloudPlatform/guest-logging-go/logger"
+)
+
+const (
+	contentDirPrefix  = "/run/secrets/workload-spiffe-contents"
+	tempSymlinkPrefix = "/run/secrets/workload-spiffe-symlink"
+	symlink           = "/run/secrets/workload-spiffe-credentials"
+)
+
+var (
+	programName    = "gce_workload_certs_refresh"
+	version        string
+	metadataURL    = "http://169.254.169.254/computeMetadata/v1/"
+	defaultTimeout = 2 * time.Second
+)
+
+func logFormat(e logger.LogEntry) string {
```

Comment:
is this logging consistent with whatever we have elsewhere in the guest-agent?

Reply:
yes this is another repo norm

Reply:
ok